README.md

```diff
@@ -234,14 +236,20 @@ If you already use other parser (e.g. `"parser": "babel-eslint"`), please move i
 
 The `vue-eslint-parser` uses the parser which is set by `parserOptions.parser` to parse scripts.
 
-### Can my javascript code have increased indentation?
+### Why don't work on `.vue` file?
 
-It depends on the version of eslint you're using.
+1. Make sure you don't have `eslint-plugin-html` in your config. The `eslint-plugin-html` extract the inside of `<script>` tags from the source code, but `eslint-vue-plugin` requires `<script>` tags and `<template>` tags in order to find script and template.
```

Comment:
1. `extracts the content from <script> tags, but ...`
2. `in order to distinguish template and script in single file components`